dcc-portal-pql/src/main/java/org/dcc/portal/pql/es/visitor/CreateQueryBuilderVisitor.java

```diff
@@ -56,14 +60,34 @@ public QueryBuilder visitQuery(@NonNull QueryNode node, @NonNull Optional<QueryC
 
   @Override
   public QueryBuilder visitNested(@NonNull NestedNode node, @NonNull Optional<QueryContext> context) {
-    val scoreQuery = node.getFirstChild().accept(this, context);
+    if (node.getFirstChild() instanceof TermsNode) {
+      val termsNode = (TermsNode) node.getFirstChild();
+      return QueryBuilders
+          .nestedQuery(node.getPath(), QueryBuilders.filteredQuery(QueryBuilders.matchAllQuery(),
+              FilterBuilders.termFilter(termsNode.getField(), ((TerminalNode) termsNode.getFirstChild()).getValue())));
```

Comment:
`TermsNode` may have multiple values. Here only first value is used. Are we sure there will be only one value?